au3/libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1106,30 +1112,31 @@ Track::Holder WaveTrack::Copy(double t0, double t1, bool forClipboard) const
          continue;
       else if (t0 <= pClip->GetPlayStartTime() && t1 >= pClip->GetPlayEndTime())
       {
-         newTrack->CopyWholeClip(*pClip, t0, forClipboard);
+         newTrack->CopyWholeClip(*pClip, t0, forClipboard, backup);
       }
       else if (pClip->CountSamples(t0, t1) >= 1) {
-         newTrack->CopyPartOfClip(*pClip, t0, t1, forClipboard);
+         newTrack->CopyPartOfClip(*pClip, t0, t1, forClipboard, backup);
       }
    }
    newTrack->FinishCopy(t0, t1, forClipboard);
    return newTrack;
 }
 
-void WaveTrack::CopyWholeClip(const Interval &clip,
-   double t0, bool forClipboard)
+void WaveTrack::CopyWholeClip(
+   const Interval& clip, double t0, bool forClipboard, bool backup)
```

Comment:
But I'm not sure that they are 😬 See
>    // If for the clipboard, then this is definitely not a backup action.
>   // MH: And if it isn't ?..

I guess this method could be called programmatically (hence not for the clipboard) but yet not for backup purpose. For this reason I prefer this uncertainty to be reflected here in case someone in the future this turns out to be a problem and someone debugs this.